releases/sentry.yaml

```diff
@@ -343,3 +346,26 @@ releases:
               requests:
                 memory: '{{ env "SENTRY_HOOKS_RESOURCES_REQUEST_MEMORY" | default "3200Mi" }}'
 
+  - name: 'sentry-alerts'
+    chart: "cloudposse-incubator/monochart"
+    version: "0.18.4"
+    wait: true
+    force: true
+    installed: {{ env "SENTRY_ALERT_INSTALLED" | default "true" }}
```

Comment:
Seems like it should be `SENTRY_ALERTS_INSTALLED` for consistency with the release name